Leanplum-SDK/Classes/LPMessageTemplates.m

```diff
@@ -94,31 +84,29 @@ - (id)init
 // Please give us suggestions for other types of messages!
 - (void)defineActions
 {
-
-    [[[LPAlertMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPConfirmMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPInterstitialMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPWebInterstitialMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPOpenUrlMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPPushAsktoAskMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPRegisterForPushMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPCenterPopupMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPHtmlMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPAppRatingMessageTemplate alloc] init] defineActionWithContexts:_contexts];
-    [[[LPIconChangeMessageTemplate alloc] init] defineActionWithContexts:_contexts];
+    [LPCenterPopupMessageTemplate defineAction];
+    [LPInterstitialMessageTemplate defineAction];
+    [LPWebInterstitialMessageTemplate defineAction];
+    [LPRichInterstitialMessageTemplate defineAction];
+    [LPOpenUrlMessageTemplate defineAction];
+    [LPRegisterForPushMessageTemplate defineAction];
+    [LPAppRatingMessageTemplate defineAction];
+    [LPIconChangeMessageTemplate defineAction];
+    [LPPrePushMessageTemplate defineAction];
+    [LPAlertMessageTemplate defineAction];
 }
 
 // If notification were enabled by Leanplum's "Push Ask to Ask" or "Register For Push",
 // refreshPushPermissions will do the same registration for subsequent app sessions.
 // refreshPushPermissions is called by [Leanplum start].
 - (void)refreshPushPermissions
 {
-    [[[LPPushAsktoAskMessageTemplate alloc] init] refreshPushPermissions];
+    [[[LPPrePushMessageTemplate alloc] init] refreshPushPermissions];
```

Comment:
If we want. to change the name here, we need to follow it up with changing this on Android + communicating the change of name to the org + documentation

Reply:
We can revert it, its just internal change to better describe what is the purpose of the template.